src/main/java/practice/StreamPractice.java

```diff
@@ -35,7 +52,10 @@ public Double getOddNumsAverage(List<Integer> numbers) {
      * Example: select men who can be recruited to army (from 18 to 27 years old inclusively).
      */
     public List<Person> selectMenByAge(List<Person> peopleList, int fromAge, int toAge) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(human -> human.getAge() >= fromAge
+                        && human.getAge() <= toAge && human.getSex().equals(Person.Sex.MAN))
+                .collect(Collectors.toList());
```

Comment:
```suggestion
                .toList();
```
Let’s  apply it to all methods